src/components/conference/NewConferenceModal/NewConferenceModal.svelte

```diff
@@ -0,0 +1,115 @@
+<script>
+    import {onMount} from 'svelte';
+    import { fade } from 'svelte/transition';
+    import IconButton from "@smui/icon-button";
+    import Form from "./Form.svelte";
+    import {displayNewConferenceModal, pending} from "./newConferenceStores.ts";
+
+
+    function closeModal() {
+        displayNewConferenceModal.set(false);
+    }
+
+    const formId = "newConferenceForm";
+    const closeButtonId="newConferenceCloseButton"
+
+    // Hack: https://stackoverflow.com/questions/45018047/prevent-background-items-from-receiving-focus-while-modal-overlay-is-covering-th
+    function createTabCycle (first, last) {
+        first.addEventListener('keydown', function(e) {
+            if (e.key==='Tab' && e.shiftKey) {
+                last.focus();
+                e.preventDefault();
+            }});
+        last.addEventListener('keydown', function(e){
+            if (e.key==='Tab') {
+                first.focus();
+                e.preventDefault();
+            }});
+    }
+
+    let firstElement, lastElement;
+
+    onMount(() => {
+        firstElement = document.querySelector("#"+formId+" .labeled-field .textfield input");
+        lastElement = document.getElementById(closeButtonId);
+        createTabCycle(firstElement, lastElement);
+        firstElement.focus();
+    })
+</script>
+
+
+<div
+        class="modal"
+        transition:fade={{duration:100}}
+        role="dialog"
+        aria-modal="true"
+>
+    <div class="modal-box">
+        <h2>Registrer en ny konferanse</h2>
```

Comment:
Mhmm ja jeg ser det. Det er ingen anbefaling fra WCAG sin side, men de fleste accessibility checker vil klage på at h1 mangler når modalen åpne. I den kursen jeg har tatt så står det bare at:
 

> With few exceptions, most web pages should have only one heading level 1, to make it clear where the beginning of the main content is.
> 
> A common exception to this best practice is when a page has a modal or overlay. The modal/overlay is an element displayed on top of the main page, and the heading level of the modal title is often marked as an <h1>. This should not cause users confusion, however, because the modal and its heading should not be available while a user is interacting with the main page, and the main page should not be available while a user is interacting with the modal/overlay.

Reply:
Mhmm ja jeg ser det. Det er ingen anbefaling fra WCAG sin side, men de fleste accessibility checker vil klage på at h1 mangler når modalen åpne. I den kursen jeg har tatt så står det bare at:

With few exceptions, most web pages should have only one heading level 1, to make it clear where the beginning of the main content is.

A common exception to this best practice is when a page has a modal or overlay. The modal/overlay is an element displayed on top of the main page, and the heading level of the modal title is often marked as an <h1>. This should not cause users confusion, however, because the modal and its heading should not be available while a user is interacting with the main page, and the main page should not be available while a user is interacting with the modal/overlay

Når WCAG ikke sier noe ting om heading i modal så kan vi lar det være :D